packages/flutter/test/material/scrollbar_test.dart

```diff
@@ -1051,8 +1051,9 @@ void main() {
           ),
           child: Scrollbar(
             controller: controller,
-            child: const SingleChildScrollView(
-              child: SizedBox(width: 4000.0, height: 4000.0),
+            child: SingleChildScrollView(
+              controller: controller,
+              child: const SizedBox(width: 4000.0, height: 4000.0),
```

Comment:
NICE

Reply:
No problem! What kind of test are you thinking of? Actually, I had to add the small extra code so that the current Cupertino tests pass. Maybe the current tests are enough?